tests/test_isrFunctions.py

```diff
@@ -0,0 +1,402 @@
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+import unittest
+import numpy as np
+
+import lsst.afw.image as afwImage
+import lsst.utils.tests
+import lsst.ip.isr as ipIsr
+import lsst.pex.exceptions as pexExcept
+import lsst.ip.isr.isrMock as isrMock
+
+
+def maskCountPixels(maskedImage, maskPlane):
+    r"""Function to count the number of masked pixels of a given type.
+
+    Parameters
+    ----------
+    maskedImage : `lsst.afw.image.MaskedImage`
+        Image to measure the mask on.
+    maskPlane : `str`
+        Name of the mask plane to count
+
+    Returns
+    -------
+    nMask : `int`
+        Number of masked pixels.
+    """
+    bitMask = maskedImage.getMask().getPlaneBitMask(maskPlane)
+    isBit = maskedImage.getMask().getArray() & bitMask > 0
+    numBit = np.sum(isBit)
+    print("%s %d %d\n" % (maskPlane, bitMask, numBit))
+    return numBit
+
+
+def imageMedianStd(image):
+    r"""Function to calculate median and std of image data.
+
+    Parameters
+    ----------
+    image : `lsst.afw.image.Image`
+        Image to measure statistics on.
+
+    Returns
+    -------
+    median : `float`
+        Image median.
+    std : `float`
+        Image stddev.
+    """
+    median = np.nanmedian(image.getArray())
+    std = np.nanstd(image.getArray())
+    print("ms: %f %f" % (median, std))
+    return (median, std)
+
+
+class IsrFunctionsCases(lsst.utils.tests.TestCase):
+    r"""Test functions for ISR produce expected outputs.
+    """
+    def setUp(self):
+        self.inputExp = isrMock.TrimmedRawMock().run()
+        self.mi = self.inputExp.getMaskedImage()
+
+    def tearDown(self):
+        pass
+
+    def test_transposeMaskedImage(self):
+        r"""Image transpose.
+
+        Expect height and width to be exchanged.
+        """
+        transposed = ipIsr.transposeMaskedImage(self.mi)
+        self.assertEqual(transposed.getImage().getBBox().getHeight(),
+                         self.mi.getImage().getBBox().getWidth())
+        self.assertEqual(transposed.getImage().getBBox().getWidth(),
+                         self.mi.getImage().getBBox().getHeight())
+
+    def test_interpolateDefectList(self):
+        r"""Defect interpolation.
+
+        Expect number of interpolated pixels to be non-zero.
```

Comment:
Again, this statement should be the entirety of the docstring: the first part doesn't add anything. Similarly for the other test docstrings below.